### DISCUSSION TITLE
New wireless password

### DISCUSSION THREAD

**Discussion by Enjoy-Combi** [arendst/Tasmota#14505] [Q&A] [answered]
Hi,

I have something like 15 devices under tasmota and today I have discovered two devices  ssid ThingsTurn_XXXX. into my network Those two devices have an irc channel open. As I have absolutely no idea what are those devices I changed the wireless password.
 Some devices under tasmota are available back on the list of access point but just many, not all of them. Do you know why ?

FYI, all devices are under version 10.1.0

**Comment by ascillato** [comment, marked as answer]
If your devices are not connected to your new wifi network it is because they don't have the correct credentials. Use the AP mode of those devices to reconfigure them. To turn the AP mode, press button1 6 times in a row (https://tasmota.github.io/docs/Buttons-and-Switches/#multi-press-functions)

**Reply by Enjoy-Combi** [reply]
Right, I can do that. I was wondering why some devices automatically enter in AP mode when they detect crednetials are wrong and not others.

**Reply by barbudor** [reply]
> why some devices automatically enter in AP mode when they detect crednetials are wrong and not others.

Check `WifiConfig`

**Reply by Enjoy-Combi** [reply]
You did it.
For some devices WifiConfig = 2, some others = 4.
Has to set this correctly.

Many thanks.
